lib/data/models/leaves/load_apply_leave_screen_model.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:convert';
+
+LoadApplyLeaveScreenModel loadApplyLeaveModelFromJson(String str) => LoadApplyLeaveScreenModel.fromJson(json.decode(str));
+
+String loadApplyLeaveModelToJson(LoadApplyLeaveScreenModel data) => json.encode(data.toJson());
+
+class LoadApplyLeaveScreenModel {
+  int status;
+  String message;
+  ApplyLeaveData data;
+
+  LoadApplyLeaveScreenModel({
+    required this.status,
+    required this.message,
+    required this.data,
+  });
+
+  factory LoadApplyLeaveScreenModel.fromJson(Map<String, dynamic> json) => LoadApplyLeaveScreenModel(
+    status: json["status"],
+    message: json["message"],
+    data: ApplyLeaveData.fromJson(json["data"]),
```

Comment:
no null handling, what is status, message, data is null?

Reply:
done